src/AdminConsole/Billing/Configuration/FeaturesOptions.cs

```diff
@@ -6,6 +6,18 @@ public class FeaturesOptions
 
     public int EventLoggingRetentionPeriod { get; set; }
 
+    /// <summary>
+    /// Maximum monthly limit for magic link emails sent by this application.
+    /// The actual limit may be lower, depending on the age of the application.
+    /// </summary>
+    public int MagicLinkEmailMaxMonthlyLimit { get; set; }
+
+    /// <summary>
+    /// By-minute rate limit for magic link emails sent by this application.
+    /// The actual limit may be lower, depending on the age of the application.
+    /// </summary>
+    public int MagicLinkEmailMaxMinutelyLimit { get; set; }
```

Comment:
I'm wondering if we should rename this to be called Rate instead of Minutely?

Reply:
It made sense to name those two properties similarly because they work essentially the same way, with the only difference being the time window